news/2024/2024-10-09-resurrection-cup-2024-results.md

```diff
@@ -0,0 +1,415 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/12840110" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Phreel.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/12840110">Phreel</a></p>
+
+Do you see the community fully switching to custom overlay solutions from osu!(lazer) for tournament streaming?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/7696512" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Hoaq.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/7696512">Hoaq</a></p>
+
+Probably no. Transitioning to custom tournament overlay means that you will need someone with coding knowledge (HTML, CSS and JavaScript at minimum). Not many tournaments have voluntary staff that can also deal with the mess of coding what essentially is a static website. But some tournaments do, and they try to go for the best look that suits their tournaments' design.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8266808" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-[Boy]DaLat.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/8266808">[Boy]DaLat</a></p>
+
+Despite being a 4 years overlay developer as I am, I am pretty sure that the community will not be switching fully to tosumemory overlays anytime soon in the future. osu!(lazer) is a native, highly maintained, fully tested, easy to setup streaming client. Even without the highly customizable experience of tosumemory overlays, it is the de facto, the one to go for when you first think of organizing a tournament. Tosumemory overlays are great, but for it to have the native-like experience of osu!(lazer) will need a lot of efforts. Each one has its own pros and cons respectively so it is best to choose which kind of client do you want based on your needs, not to be biased by any of the majority.
+
+</div>
+
+---
+
+In the tournament history of osu!, many people have hosted successful tournaments, whether it be for player enjoyment or viewer enjoyment. Regardless of the goal, the pressure stays the same. In our case, the tiny community that developed itself over the past 4 ResCup tournaments certainly elevated vast majority of this pressure and made even the preparations not only stress free, but a joy. Thank you to each and every single person who worked on Resurrection Cup 2024, and thank you to everyone who supports us! See you all next year, in **Resurrection Cup 2025: Regulus Revolution**.
```

Comment:
is it 4 or 3 iterations? the first sentence in the Serenity bits says there were 3 past iterations, with the below graphic including the latest one among the 3
```suggestion
In the tournament history of osu!, many people have hosted successful tournaments, whether it be for player enjoyment or viewer enjoyment. Regardless of the goal, the pressure stays the same. In our case, the tiny community that developed itself over the past 4 ResCup tournaments certainly elevated vast majority of this pressure and made even the preparations not only stress-free, but a joy.

Thank you to each and every single person who worked on Resurrection Cup 2024, and thank you to everyone who supports us! See you all next year, in **Resurrection Cup 2025: Regulus Revolution**.
```